src/components/posts/view/postsView.js

```diff
@@ -47,11 +47,12 @@ class PostsView extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { currentAccountUsername } = this.props;
+    const { currentAccountUsername, changeForceLoadPostState } = this.props;
```

Comment:
func name is not valuable

Reply:
?

Reply:
`changeForceLoadPostState` what its changing what is working for this function. I dont understand from name

Reply:
it is changing to ForceLoadPost state